WordPress/Classes/Models/WPAccount.m

```diff
@@ -52,6 +52,10 @@ - (void)didTurnIntoFault
 
 - (NSString *)password
 {
+    if (self.isWpcom) {
+        return [NSString string];
```

Comment:
I was trying to figure out why we need to return an empty `NSString` in this case, but couldn't really figure it out.  Can you clarify?

My main concern is that internally we tend to use `nil` for unset properties - and this would go out of the norm a bit, which I would avoid if possible.  If some interface needs a string instead of a nil value, I believe it should be that interface to make the appropriate conversion.  Let me know.